packages/react-native-bpk-component-phone-input/src/__snapshots__/BpkDialingCodeList-test.android.js.snap

```diff
@@ -103,7 +103,7 @@ exports[`Android BpkDialingCodeList should render correctly 1`] = `
                   "width": 24,
                 },
                 Array [
-                  null,
+                  undefined,
```

Comment:
These look a bit dodgy - we should find out why `undefined` was added and if it's intentional

Reply:
It seems this has come in since we have wrapped the `BpkFlag` with the `withTheme` - if we change `withTheme(BpkFlag)` to `BpkFlag` then it changes back to `null`.